src/test/kotlin/graphql/kickstart/tools/DirectiveTest.kt

```diff
@@ -244,11 +244,6 @@ class DirectiveTest {
         val name: String
     )
 
-    private enum class AllowedState {
-        ALLOWED,
-        DISALLOWED
-    }
-
```

Comment:
This enum was unused.

Reply:
It's not used because the test is disabled for now. Should probably add it back.

Reply:
Right you are, adding back.